episodes/excthreesteps.Rmd

```diff
@@ -429,7 +429,7 @@ In this step, we visualise the final results of the image registration process.
 
 ## 11. Preparing Images for calling the demonsReg function
 
-Follow `exampleSolution3.py` for running the function. Below, we describe how we prepare images: `cine_MR_img_1.png`, `cine_MR_img_2.png`, and `cine_MR_img_3.png`. These images will be available in Data folder. 
+Follow `exampleSolution3.py` [available in src/mirc/Example] for running the function. Below, we describe how we prepare images: `cine_MR_img_1.png`, `cine_MR_img_2.png`, and `cine_MR_img_3.png`. These images are available in `episodes/data`. 

```

Comment:
Hi @zsenousy 

Can you please rename `Example` path as `examples` using lowercase (also in the actual paths), as we can use such directory for other examples. Why lowercases? "Modules should have short, all-lowercase names https://peps.python.org/pep-0008/#package-and-module-names" 

Thanks, Miguel